samples/frameworks/react/ts/README.md

```diff
@@ -0,0 +1,54 @@
+# Azure SDK samples for React (TypeScript)
+
+This sample application shows how to use the TypeScript client libraries for Azure in some common scenarios.
+
+In this sample, we build a simple Todo application in React using [create-react-app][react] and integrating with various Azure services.
+
+- Integration with Azure EventHubs showcases a mechanism for multiple instances to consume EventHub messages produced by any one of them.
+- Integration with Azure ServiceBus showcases a mechanism for any single instance to consume ServiceBus messages. When multiple windows are open, only one of them will receive any given message.
+- Integration with Azure Storage Blob showcases a mechanism for fetching Azure blobs and displaying them in the browser.
+
+## Prerequisites
+
+The samples are compatible with Node.js >= 8.0.0.
+
+Before running the samples in Node, they must be compiled to JavaScript using the TypeScript compiler. For more information on TypeScript, see the [TypeScript documentation][typescript].
+
+You need [an Azure subscription][freesub] and the following resources created to run this sample:
+
+- An Azure EventHubs namespace. Please refer to the [EventHubs documentation][eventhubs] for additional information on EventHubs
```

Comment:
In case anyone else runs into this.. 

This was caused because I had connection string in the env variable when the expected was the namespace endpoint.
`<namespace>.servicebus.windows.net`.

Reply:
Thanks @maorleger